frontend/README.md

```diff
@@ -1,114 +1,116 @@
+
+  ![View list picture](docs/Images/View_page.png)
+
+The frontend for this project was built with the following libraries and frameworks:
+
+- [TypeScript](https://www.typescriptlang.org/)
+- [Vue.js](https://github.com/vuejs/vue) / [Vue CLI](https://github.com/vuejs/vue-cli)
+- [Vuelidate](https://github.com/vuelidate/vuelidate)
+- [Axios](https://github.com/axios/axios)
+
+Other useful tools
+
+- [Visual Studio Code](https://code.visualstudio.com/?WT.mc_id=theurlist-github-buhollan)
+- [Vetur](https://marketplace.visualstudio.com/items?itemName=octref.vetur&WT.mc_id=theurlist-github-buhollan)
+- [VS Code Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome&WT.mc_id=theurlist-github-buhollan)
+- [Vue VS Code Extension Pack](https://marketplace.visualstudio.com/items?itemName=sdras.vue-vscode-extensionpack&WT.mc_id=theurlist-github-buhollan)
+- [Vue browser devtools](https://github.com/vuejs/vue-devtools)
+
+## <a name="feconfigs" ></a> Frontend configurations
+
+There are two configurations needed for the frontend to run, those are passed as [VUE environment variables](https://cli.vuejs.org/guide/mode-and-env.html)
+
+- VUE_APP_BACKEND: this has the URL pointing to the backend endpoint.
+- VUE_APP_FRONTEND: this has the URL point to the frontend endpoint.
+- VUE_APP_OIDC_AUTHORITY: the configured Azure B2C Open ID endpoint.
+- VUE_APP_OIDC_CLIENT_ID: the configured Azure B2C Client Application ID.
+- VUE_APP_OIDC_SCOPE: the Azure B2C Client Scope.
+
+## Build and run the frontend locally
+
+### Modify Environment Variable
+
+- Follow the guide on how to run the Backend locally. You can find the README [here](../api/README.md)
+- Once the Backend has started, you will want to get the port the Backend uses _(local port may change depending upon the IDE being used)_.
+- Create a local configuration file called `env.development` and add the needed configs as described [above](feconfigs), something like the following:
+
+```bash
+VUE_APP_BACKEND=http://localhost:[backend_port]
+VUE_APP_FRONTEND=http://localhost:[frontend_port]
+VUE_APP_OIDC_AUTHORITY=[Azure B2C Open ID config endpoint]
+VUE_APP_OIDC_CLIENT_ID=[Azure B2C Client Application ID]
+VUE_APP_OIDC_SCOPE=[Azure B2C Client Scope]
+```
+
+### Install Vue CLI globally
+
+```bash
+npm install -g @vue/cli
+```
+
+### Install npm packages for frontend project
+
+```bash
+npm install
+```
+
+### Serve development build
+
+```bash
+npm run serve
+```
+
+![localhost serve](docs/Images/localhost_serve.png)
+
+### Create production build
+
+```bash
+npm run build
+```
+
+_This creates a dist folder under frontend_
+
+### Lints and fixes files
+
+```bash
+npm run lint
+```
+
+## Debugging the application
+
+- Follow the instructions in the [Build and run the frontend locally](##-build-and-run-the-frontend-locally) to start application
+- In Chrome, press `F12` to start the Chrome Developer Tools

```

Comment:
Let's revise and say "open the Developer tools." It's not the same hot-keys for all of the OS.